### PR TITLE
Feature/create new agents configuration

### PR DESCRIPTION
**Pull request by dcorquir**
Added endpoint to create and update agents configuration

### CODE REVIEW

File: src/models/general/simulation.py [hunk 1]
```diff
@@ -0,0 +1,8 @@
+from enum import Enum
+
+
+class SimulationStatus(Enum):
+    INCOMPLETE: str = 'incomplete'
+    DONE: str = 'done'
+    RUNNING: str = 'running'
+    ERROR: str = 'error'
```
Comment (davisfelipe) [inline src/models/general/simulation.py:8]: Line Break at the end of the file

File: src/models/db/configuration.py [hunk 2]
```diff
@@ -0,0 +1,22 @@
+from datetime import datetime
+from mongoengine import (
```
Comment (davisfelipe) [inline src/models/db/configuration.py:2]: Sort Imports [PEP8 Import](https://www.python.org/dev/peps/pep-0008/#imports)
1. Standard library imports.
2. Related third party imports.
3. Local application/library specific imports.

File: src/models/__init__.py [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+from .db.Simulation import Simulation
+from .route_models.configuration import NewConfiguration
+
+__all__ = [
+    "Configuration", 
+    "DiseaseStates", 
+    "Distribution", 
+    "Simulation",
+    "NewConfiguration"
+]
```
Comment (davisfelipe) [inline src/models/__init__.py:13]: Line Break at the end of the line

File: src/interfaces/tests/test_configuration_interface.py [hunk 4]
```diff
@@ -0,0 +1,57 @@
+import uuid
+
+from unittest import TestCase
+
+from mongoengine import connect, disconnect
+
+from src.interfaces import ConfigurationInterface
+from src.models import Configuration
```
Comment (davisfelipe) [inline src/interfaces/tests/test_configuration_interface.py:8]: Sort Imports

File: src/interfaces/disease_states_interface.py [hunk 5]
```diff
@@ -0,0 +1,12 @@
...
+class DiseaseStatesInterface:
+    '''
+        Interface to consult disease states in DB
+    '''
+    @staticmethod
+    def fin_all():
+        """
+        Find all disease states
+        """
+        return MasterDiseaseStates.objects().all()
```
Comment (davisfelipe) [inline src/interfaces/disease_states_interface.py:12]: Line break at the end of the line

File: src/interfaces/configuration_interface.py [hunk 6]
```diff
@@ -0,0 +1,51 @@
...
+
+    @staticmethod
+    def find_all():
+        """
+        Get all configurations exists in db
+
+        Return:
+            Configuration objects as list
+        """
+        return Configuration.objects().objects().all()
```
Comment (davisfelipe) [inline src/interfaces/configuration_interface.py:17]: Why do you call this object twice?

**Review by davisfelipe — CHANGES_REQUESTED** [review]
Hi Diego, I made comments on your code.

Remeber we use [reST Docstring](https://www.python.org/dev/peps/pep-0287) format

**Comment by Camilo-HG** [comment]
@davisfelipe aren't we using [numpy docs](https://numpydoc.readthedocs.io/en/latest/format.html) ???

**Comment by dcorquir** [comment]
> 
> 
> @davisfelipe aren't we using [numpy docs](https://numpydoc.readthedocs.io/en/latest/format.html) ???

Yes we're using numpydocs.

**Comment by Camilo-HG** [comment]
But you said: "Remeber we use reST Docstring format"

**Comment by dcorquir** [comment]
> 
> 
> But you said: "Remeber we use reST Docstring format"

Yes, but on code comments we're using numpy structure.

**Comment by davisfelipe** [comment]
We didn't use Numpy Doc, because:

1. Numpy is better for libraries or independent components, in API Rest we used to write reST Docstring, because is short and precise, Numpy is very extensive for a simple comment.
3. PEP8 Recommend use this format, it is very intuitive. 
2. In FastAPI endpoints, we use reST Docstring to describe endpoints and it information show in OpenAPI and ReDoc.

**Comment by Camilo-HG** [comment]
Ok. I got it.

File: src/routes/configuration.py [hunk 7]
```diff
@@ -0,0 +1,115 @@
...
+from src.utils.response import UJSONResponse
+
+
+configuration_routes = APIRouter(tags=["Configuration"])
+
+
+@configuration_routes.get("/configuration")
+def find_all():
+    """
+    Get all configuration exist in db
```
Comment (Camilo-HG) [inline src/routes/configuration.py:19]: Change: "Get all configurations exists in db" ---> "Get all existing configurations in db"

**Review by Camilo-HG — CHANGES_REQUESTED** [review]

**Pull request merged**